setup.py

```diff
@@ -7,14 +7,10 @@
 
 HERE = path.abspath(path.dirname(__file__))
 
-with open_f(path.join(HERE, "DESCRIPTION.rst"), encoding="utf-8") as f:
```

Comment:
The proper way to do it would be to wrap this context manager into a try/except IOError block that sets LONG_DESCRIPTION to an empty string when the file cannot be found. There's a `Makefile` with a target that generates the .rst file from the README, using Pandoc.